source/puppet/latest/cli.md

```diff
@@ -1,12 +1,12 @@
 ---
 layout: default
-built_from_commit: d3ee2f44b7cb0cadbb6fc9495f2eb093c4d6e148
+built_from_commit: b1ccc5d8d335a57108919a17a10d0ab80e91b2fc
 title: 'Facter: CLI'
 toc: columns
 canonical: "/puppet/latest/cli.html"
 ---
 
-# Facter CLI
+# TEST PREABLE
```

Comment:
Hi @IrimieBogdan! Could you please change the name of this in the source file? I can update it manually in the meantime.